test/e2e/set/scyllacluster/scyllacluster_pv.go

```diff
@@ -5,35 +5,272 @@ package scyllacluster
 import (
 	"context"
 	"fmt"
+	"strings"
 
 	g "github.com/onsi/ginkgo/v2"
 	o "github.com/onsi/gomega"
 	scyllav1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1"
 	"github.com/scylladb/scylla-operator/pkg/controllerhelpers"
+	"github.com/scylladb/scylla-operator/pkg/helpers"
 	"github.com/scylladb/scylla-operator/pkg/naming"
 	scyllafixture "github.com/scylladb/scylla-operator/test/e2e/fixture/scylla"
 	"github.com/scylladb/scylla-operator/test/e2e/framework"
 	"github.com/scylladb/scylla-operator/test/e2e/utils"
+	"github.com/scylladb/scylla-operator/test/e2e/utils/image"
 	corev1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/tools/cache"
+	watchtools "k8s.io/client-go/tools/watch"
+	"k8s.io/utils/pointer"
 )
 
-var _ = g.Describe("ScyllaCluster Orphaned PV", func() {
-	defer g.GinkgoRecover()
-
+var _ = g.Describe("ScyllaCluster Orphaned PV controller", func() {
```

Comment:
this need to move to serial suite, it's not multitenant anymore

Reply:
I wonder whether renaming the single node we have would solve it as it would be closer to the real world - maybe we have already talked about it but it has been a while

Reply:
> this need to move to serial suite, it's not multitenant anymore
why not? It's not disturbing other tests, the same way previous one didn't